doc/api/statistics.rst

```diff
@@ -48,6 +48,13 @@ Non-parametric (clustering) resampling methods:
    summarize_clusters_stc
    bootstrap_confidence_interval
 
+ERP-related statistics:
+
```

Comment:
```suggestion
ERP-related statistics:

.. automodule:: mne.stats.erp
   :no-members:
   :no-inherited-members:
.. currentmodule:: mne.stats.erp
```

Reply:
... or something like this would work. Locally you can iteratively do `make html-noplot` (will take ~10 min the first time then should be quick on follow-up runs). I can push a commit if you want